provisioning/tasks/www.yml

```diff
@@ -88,7 +88,7 @@
 
 - name: Copy Nginx vhosts into place.
   template:
-    src: ../templates/nginx-vhost.conf.j2
+    src: "{{ nginx_vhost_template }}"
     dest: "{{ nginx_vhost_path }}/{{ item.server_name.split(' ')[0] }}.conf"
     force: yes
     owner: root
```

Comment:
@geerlingguy any chance you'd consider making your nginx role support multiple vhost files rather than looping server directives within one vhost file? Then we could remove this task :)